### ISSUE TITLE
mouse hover glitchy

### ISSUE THREAD

**Issue by miltonc90** [lwouis/alt-tab-macos#681]
Using the mouse hover seems to confuse the app and it will glitch out in a way that sends the selection quickly back and forth.  I disabled so that if the mouse happens to be hovering in the center of the screen, it doesn't create a glitch.  I'm happy to not use mouse hover, personally, but nonetheless this is a glitch.  Very thankful for this app, tho!  Exactly what I needed to switch between multiple windows of the same app without having to create new Desktops (the 4-finger swipe up on Mac)

**Comment by lwouis** [comment]
Hi @miltonc90! Thanks for sharing this issue.

Could you please share more details? I'm not understanding really what you are describing. Could you please share a video of the issue in action and steps to reproduce so i can witness it on my own machine?

Thank you 👍

**Comment by miltonc90** [comment]
I apologize, I deleted the app and changed my email.  Basically what happens is when you use the alt-tab function (which I had set to opt-tab), you engage that function and see your app icons in the center of the screen with one being highlighted.  If my mouse cursor was anywhere near an icon edge, the "highlight" would bounce between two quickly, jittering unintentionally and quite obnoxiously.  I hope that explains the problem well enough.  This is just for developer knowledge / awareness.  My only use for the app was to switch between different windows of the same app, so this was a deal breaker.  Otherwise it's a great app and that function is a great idea.  Too bad there isn't a simpler version with only that function.

macOS 10.15.7 on a late-2015 retina Macbook Pro (i5 CPU, 8GB RAM).
____________Connor Milton845-532-4538miltonc90@yahoo.com
 

    On Monday, January 18, 2021, 10:34:12 AM EST, lwouis <notifications@github.com> wrote:  
 
 


Closed #681.
